### ISSUE TITLE
Invalid version range in Maven doclint-java8-disable profile with gson 2.6.2

### ISSUE THREAD

**Issue by JulienGremillot** [google/gson#819]
Apparently, I'm running into the same problem that some people had a few versions ago (cf. https://github.com/google/gson/issues/588)

Project ID: com.google.code.gson:gson-parent
Reason: Invalid JDK version in profile 'doclint-java8-disable': Unbounded range: [1.8, for project com.google.code.gson:gson-parent

I'm running Maven 2.2.1 but it's the first time I see this kind of problem.

**Comment by inder123** [comment]
Can you submit a pull request to fix this?

**Comment by inder123** [comment]
Submitted fix in https://github.com/google/gson/pull/820

**Comment by AMill7** [comment]
Hi, i have the same error. My problem is maven-ant-tasks. It is no longer maintained and it works with jdk lower then 1.8 so because of it, it comes to a error. The only solution that i found is to use gson 2.6.1.

**Comment by LM450N** [comment]
I had the same issue and I think the error comes from `gson-parent` JDK version. The range is invalid. To fix it I overwrite the `maven-javadoc-plugin` settings.

```xml
 <!--JavaDoc profile-->
<profile>
  <id>doclint-java8-disable</id>
  <activation>
    <jdk>[1.8,)</jdk>
  </activation>
  <build>
    <plugins>
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-javadoc-plugin</artifactId>
        <configuration>
          <additionalparam>-Xdoclint:none</additionalparam>
        </configuration>
      </plugin>
    </plugins>
  </build>
</profile>
```

It simply disables the doclint for Java8. The difference with the [original settings](https://github.com/google/gson/blob/gson-parent-2.6.2/pom.xml#L108) is:
```diff
-    <jdk>[1.8,</jdk>
+    <jdk>[1.8,)</jdk>
```

This error seems fixed since the `v2.7.0`.

**Comment by Marcono1234** [comment]
Looks like this was fixed by #820; though the current `pom.xml` does not contain that specific version range anymore anyway.
